Source/WebCore/css/CSSPrimitiveValueMappings.h

```diff
@@ -5518,6 +5518,28 @@ template<> inline CSSPrimitiveValue::operator FontSynthesisLonghandValue() const
     return FontSynthesisLonghandValue::Auto;
 }
 
+template<> inline CSSPrimitiveValue::CSSPrimitiveValue(std::optional<float> sizeAdjust)
+    : CSSValue(PrimitiveClass)
+{
+    if (!sizeAdjust.has_value()) {
+        setPrimitiveUnitType(CSSUnitType::CSS_VALUE_ID);
+        m_value.valueID = CSSValueNone;
```

Comment:
There's gotta be a way to do this more elegantly.

Reply:
Hate to break it to you: there is not! But we could make one.